graylog2-server/src/main/java/org/graylog2/indexer/indices/Indices.java

```diff
@@ -211,6 +214,26 @@ public String aliasTarget(String alias) {
         return response.getAliases().isEmpty() ? null : response.getAliases().keysIt().next();
     }
 
+    public boolean createIndexTemplate() {
+        final Map<String, Object> template = indexMapping.messageTemplate(allIndicesAlias(), configuration.getAnalyzer());
+        final PutIndexTemplateRequest itr = c.admin().indices().preparePutTemplate(GRAYLOG_INTERNAL_TEMPLATE_NAME)
```

Comment:
It will just overwrite any changes with our configured values, which is what we want I guess.